src/ci.yml

```diff
@@ -53,6 +53,7 @@ jobs:
       }
       ./setup-e2e-tests.ps1 -FunctionsRuntimeVersion $env:FUNCTIONSRUNTIMEVERSION `
                             -UseCoreToolsBuildFromIntegrationTests:$useIntegrationTestingCoreTools
+                            -SkipCosmosDBEmulator
```

Comment:
This change will be removed before merging PR. It's only for testing to unblock pipeline.